RetailCoder.VBE/UI/Refactorings/Rename/RenameViewModel.cs

```diff
@@ -72,7 +72,8 @@ public bool IsValidName
                 return !(NewName.Equals(Target.IdentifierName, StringComparison.InvariantCultureIgnoreCase)) &&
                        char.IsLetter(NewName.FirstOrDefault()) &&
                        !tokenValues.Contains(NewName, StringComparer.InvariantCultureIgnoreCase) &&
-                       !NewName.Any(c => !char.IsLetterOrDigit(c) && c != '_');
+                       !NewName.Any(c => !char.IsLetterOrDigit(c) && c != '_') &&
+                       NewName.Length <= (Target.DeclarationType.HasFlag(DeclarationType.Module) ? 31 : 255);
```

Comment:
magic numbers?

Reply:
Yup.

Reply:
They could be public constants in `Declaration`, no?  `MaxLengthModuleName` and `MaxLengthMemberName`?

Reply:
Cool.